packages/wrangler/src/__tests__/helpers/mock-oauth-flow.ts

```diff
@@ -174,7 +174,23 @@ export const mockOAuthFlow = () => {
 		);
 	};
 
+	const mockDomainUsesAccess = ({ usesAccess }: { usesAccess: boolean }) => {
+		// If the domain relies upon Cloudflare Access, then a request to the domain
+		// will result in a redirect to the `cloudflareaccess.com` domain.
+		fetchMock.mockOnceIf("https://dash.cloudflare.com/", async () => {
```

Comment:
I wonder if this domain should be configurable so tests can be added for other domains that use access?

Reply:
Hmm... OK. I will add tests for the case where we override the URL via the env variable. Thanks for the prompt.

Reply:
It turned out to be a bit more work than I expected: https://github.com/cloudflare/wrangler2/pull/1869/commits/d825e03430ba52fd1e663a1781ddeb816d62739b

PTAL @penalosa